spec/mailers/notifier_spec.rb

```diff
@@ -105,6 +105,32 @@
     end
   end
 
+  describe ".mentioned limited" do
+    before do
+      @user = alice
+      @post = FactoryGirl.create(:status_message, public: false)
+      @mention = Mention.create(:person => @user.person, :post => @post)
+
+      @mail = Notifier.mentioned(@user.id, @post.author.id, @mention.id)
+    end
+
+    it 'TO: goes to the right person' do
+      expect(@mail.to).to eq([@user.email])
+    end
+
+    it 'SUBJECT: has the name of person mentioning in the subject' do
+      expect(@mail.subject).to include(@post.author.name)
+    end
+
+    it 'has the post text not in the body' do
+      expect(@mail.body.encoded).not_to include(@post.text)
+    end
+
+    it 'should not include translation fallback' do
+      expect(@mail.body.encoded).not_to include(I18n.translate 'notifier.a_post_you_shared')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.